config/install/atm.settings.yml

```diff
@@ -1,8 +1,13 @@
 register_url: https://www.adtechmedia.io/admin/accounts/signup
```

Comment:
Please create an `atm.settings.dev.yml` and load it with all development endpoints when localhost used